### PR TITLE
Use b4a

### PR DESCRIPTION
**Pull request by Nazeh**
As it says on the tin.

### CODE REVIEW

File: lib/keys.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 const sodium = require('sodium-universal')
 const blake2b = require('blake2b-universal')
+const Buffer = require('b4a')
```
Comment (mafintosh) [inline lib/keys.js:5]: name this b4a for clarity (not a buffer)

**Comment by mafintosh** [comment]
@kasperisager could you take a look at this? you're the b4a expert

**Review by kasperisager — APPROVED** [review]
LGTM! 👍

**Comment by andrewosh** [comment]
Thanks @Nazeh @kasperisager -- released as 6.0.1-alpha.10 (`corestore@next`).

**Pull request merged**